src/login.rs

```diff
@@ -1002,34 +1006,39 @@ impl Login {
     }
 
     fn show_no_email_access_dialog(&self) {
-        let dialog = gtk::MessageDialog::builder()
-            .text(&gettext("Sorry"))
-            .secondary_text(&gettext(
+        let dialog = adw::MessageDialog::builder()
+            .heading(&gettext("Sorry"))
+            .body(&gettext(
                 "If you can't restore access to the e-mail, your remaining options are either to remember your password or to delete and then recreate your account.",
             ))
-            .buttons(gtk::ButtonsType::Close)
-            .modal(true)
             .transient_for(self.root().unwrap().downcast_ref::<gtk::Window>().unwrap())
             .build();
 
-        dialog.add_button(&gettext("_Go Back"), gtk::ResponseType::Other(0));
-
-        dialog.run_async(clone!(@weak self as obj => move |dialog, response_id| {
-            dialog.close();
+        dialog.add_responses(&[
+            ("close", &gettext("_Close")),
+            ("go-back", &gettext("_Go Back")),
```

Comment:
Since we're also improving UX a bit, I'd say to remove both of these and just leave an "OK" response that just closes the dialog. I find "Close" and "Go Back" to be quite confusing as it seems that they do the same thing.

Reply:
You mean it could give the impression that they do the same? Because `Go Back` actually brings the user back to the `password-forgot-page`.

If we just leave one `OK`, I think the action row should be exchanged with something else, as it does not initiate a view transition anymore.
A pill button, maybe.

Reply:
> You mean it could give the impression that they do the same?

Yeah, exactly.

> If we just leave one OK, I think the action row should be exchanged with something else, as it does not initiate a view transition anymore.
A pill button, maybe.

I think a `GtkLinkButton` is a better fit for this (it's also similar to the ToS case).

Reply:
![Bildschirmfoto vom 2022-09-11 11-45-53](https://user-images.githubusercontent.com/3630213/189521395-ff90d731-f846-4a5d-826a-9b1c60dfa870.png)
![Bildschirmfoto vom 2022-09-11 11-46-21](https://user-images.githubusercontent.com/3630213/189521396-a11e8776-6558-44ca-8307-442769c4bb5a.png)